src/item.cpp

```diff
@@ -6571,8 +6547,14 @@ void item::heat_up()
     item_tags.erase( "FROZEN" );
     item_tags.erase( "WARM" );
     item_tags.insert( "HOT" );
-    // links the amount of heat an item can retain to its mass
-    item_counter = clamp( to_gram( weight() ), 100, 600 );
+    // Set item temperature to 50 C (323.15 K, 122 F)
+    // Also set the energy to match
+    temperature = 122;
+    
+    float true_energy = get_energy_from_temperature( 323.15 );
+    true_energy = 2 * ( true_energy ) + 0.5;
```

Comment:
Two lines above could be compressed into one:
```suggestion
    const float true_energy = 2 * ( get_energy_from_temperature( 323.15 ) ) + 0.5;
```